backend/papersfeed/views.py

```diff
@@ -21,6 +22,7 @@ def api_not_found():
     raise ApiError(404)
 
 
+@csrf_exempt
```

Comment:
Is it ok to use @csrf_exempt about all apis? Then, how we use protection? Because in practice session, TAs mentioned about it.
<img width="997" alt="스크린샷 2019-11-02 17 08 52" src="https://user-images.githubusercontent.com/35535636/68068031-736c2d80-fd93-11e9-91b1-fffe9bacbf75.png">

Reply:
이 부분은 review_reply_api에서 작업을 했어요! 이 브랜치에서 했어야했는데 죄송합니다..

Reply:
You mean, on the branch, do you use Django's CSRF solution?

Reply:
from django.views.decorators.csrf import ensure_csrf_cookie
사용했습니다

Reply:
Ok, I'll expect the PR from that branch include those parts. Maybe it should be opened not too late!

Reply:
넵!